console/src/views/MultimediaPanel.js

```diff
@@ -0,0 +1,108 @@
+import {Cell, Row, Column} from '@enact/ui/Layout';
```

Comment:
```suggestion
import {Cell, Column, Row} from '@enact/ui/Layout';
```

Can you reorder the imports per our guidelines?

Reply:
I just removed Row cause it wasn't even being used.